x-pack/legacy/plugins/maps/public/layers/styles/components/vector/vector_style_editor.js

```diff
@@ -140,23 +140,20 @@ export class VectorStyleEditor extends Component {
   }
 
   _renderPointProperties() {
-    let lineColor;
-    let lineWidth;
     let iconOrientation;
-    if (this.props.styleProperties.symbol.options.symbolizeAs === SYMBOLIZE_AS_CIRCLE)  {
-      lineColor = (
-        <Fragment>
-          {this._renderLineColor()}
-          <EuiSpacer size="m" />
-        </Fragment>
-      );
-      lineWidth = (
-        <Fragment>
-          {this._renderLineWidth()}
-          <EuiSpacer size="m" />
-        </Fragment>
-      );
-    } else {
+    const lineColor = (
```

Comment:
Since `lineColor` and `lineWidth` are now constant, their definition can just be moved directly into the return block and `lineColor` and `lineWidth` variables are no longer needed.